helpers/utils.js

```diff
@@ -48,18 +51,25 @@ const sendSongMessage = (message, gameSession, isForfeit) => {
         }
     })
 }
-const sendInfoMessage = (message, title, description) => {
-    message.channel.send({
-        embed: {
-            color: EMBED_INFO_COLOR,
-            author: {
-                name: message.author.username,
-                icon_url: message.author.avatarURL()
-            },
-            title: `**${title}**`,
-            description: description
+const sendInfoMessage = (message, title, description, footerText, footerImageWithPath) => {
+    let footerImage = footerImageWithPath ? path.basename(footerImageWithPath) : null;
+    let embed = new Discord.MessageEmbed({
+        color: EMBED_INFO_COLOR,
+        author: {
+            name: message.author.username,
+            icon_url: message.author.avatarURL()
+        },
+        title: `**${title}**`,
+        description: description,
+        footer: {
```

Comment:
You can attach the `footer` afterwards in the if statement afterwards

Reply:
Awesome, just did it!